.github/workflows/build-and-test-pr.yml

```diff
@@ -338,3 +338,69 @@ jobs:
 #        with:
 #          name: "error-inspection-result"
 #          path: ${{ github.workspace }}/dubbo-test-tools/dubbo-error-code-inspector/error-inspection-result.txt
+
+  intergration-test:
+    needs: [check-license, build-source, unit-test-prepare]
+    name: "Intergration Test with JDK ${{ matrix.jdk }}"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        jdk: [8, 11]
+      fail-fast: false
+    env:
+      DISABLE_FILE_SYSTEM_TEST: true
+      CURRENT_ROLE: ${{ matrix.case-role }}
+    steps:
+      - name: Checkout Self
+        uses: actions/checkout@v4
+      - name: Checkout dubbo-spi-samples
+        uses: actions/checkout@v4
+        with:
+          repository: apache/dubbo-spi-samples
+          path: dubbo-spi-samples
+      - name: "Set up JDK ${{ matrix.jdk }}"
+        uses: actions/setup-java@v4
+        with:
+          distribution: "zulu"
+          java-version: ${{ matrix.jdk }}
+      - uses: actions/cache@v4
+        name: "Cache local Maven repository"
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}-${{ github.run_id }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+            ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+      - uses: actions/cache@v4
+        name: "Cache zookeeper binary archive"
+        id: "cache-zookeeper"
+        with:
+          path: ${{ github.workspace }}/.tmp/zookeeper
+          key: zookeeper-${{ runner.os }}-${{ env.ZOOKEEPER_VERSION }}
+          restore-keys: |
+            zookeeper-${{ runner.os }}-
+      - uses: actions/cache@v4
+        name: "Cache secret key"
+        id: "cache-secret-cert"
+        with:
+          path: ${{ github.workspace }}/.tmp/rsa
+          key: secret-rsa-${{ runner.os }}-${{ github.run_id }}
+      - name: Calc Dubbo Version
```

Comment:
This approach does not integrate the two test environments very well, you can check the log of the action, all tests are ignored.

Reply:
I think it's a known issue since most of spi-samples require their dubbo version to `3.2*` instad of our 3.3 which is configured in `case-versions.conf`.

Reply:
> I think it's a known issue since most of spi-samples require their dubbo version to `3.2*` instad of our 3.3 which is configured in `case-versions.conf`.

So the logic of the spi-samples repository is broken, and I think it's enough to limit the spi-extensions version. The current configuration of the samples repository is just for use, not anything special.